### PR TITLE
add providers oberschwabenmobil, gmuend_bewegt

### PR DESCRIPTION
**Pull request by ThorstenFroehlinghaus**
new moqo providers

### CODE REVIEW

File: x2gbfs/providers/moqo.py [hunk 1]
```diff
@@ -144,11 +144,13 @@ def _extract_from_parkings(
             'name': elem['name'],
             'lat': center['lat'],
             'lon': center['lng'],
-            'address': address,
-            'post_code': elem['zipcode'],
             'rental_methods': ['key'],
         }
 
+        if address:
+            station['address'] = address
```
Comment (hbruch) [inline x2gbfs/providers/moqo.py:151]: Are address / their components really optional? Above, you access them via [], are the properties contained but empty?

**Review by hbruch — APPROVED** [review]
Could you please add your changes to the CHANGELOG.md before merging.

File: x2gbfs/providers/moqo.py (hunk 1 above)
Comment (ThorstenFroehlinghaus) [reply x2gbfs/providers/moqo.py:151]: Oberschwaben Mobil has a station "RV-Nordstadt" with geocoordinates, but without any address information. The attributes town, zipcode, street, streetnumber are available, but they are = "".

File: x2gbfs/x2gbfs.py [hunk 2]
```diff
@@ -54,12 +54,16 @@ def build_extractor(provider: str, feed_config: Dict[str, Any]) -> BaseProvider:
...
+        'zeag_energie',
+        'flinkster_carsharing',
+        'oberschwabenmobil',
+        'gmuend_bewegt',
+    ]:
         return MoqoProvider(feed_config)
     if provider in ['noi']:
         return NoiProvider()
-    if provider in ['flinkster']:
-        return FlinksterProvider(feed_config)
```
Comment (hbruch) [inline x2gbfs/x2gbfs.py:62]: An oversight in the review: you removed the `flinkster` option here, but kept the import, the config and the `FlinksterProvider` itself. Either it should be removed completely or kept here as well.

**Pull request merged**